sections/exchange/TradeCard/Charts/Types/CandlesticksChart.tsx

```diff
@@ -24,43 +23,71 @@ const CandlesticksChart: FC<CandlesticksChartProps> = ({
 }) => {
 	const theme = useContext(ThemeContext);
 
-	const chartData = data.map((candle: any) => ({
-		timestamp: Number(candle.timestamp) * 1000,
-		uv: [Number(formatEther(candle.open)), Number(formatEther(candle.close))],
-		pv: [Number(formatEther(candle.high)), Number(formatEther(candle.low))],
-	}));
+	const chartData = data.map((candle: any) => {
+		const { close, high, low, open, timestamp } = candle;
+		const [day, month] = formatDate(Number(timestamp) ?? 0, 'dd MMM').split(' ');
```

Comment:
Somewhere in here it thinks the candlestick month is Jan

![image](https://user-images.githubusercontent.com/4015977/127025841-b78de062-936b-42bd-9de0-6a363df685cc.png)

Reply:
so it looks like the data coming back from the subgraphs isn't entirelyyyy uniform. For the:
- `useCombinedRates` hook, the timestamps come back as numbers in milliseconds
- `useCombinedCandlesticksChartsData` hook, the timestamps come back as strings in seconds

Just some initial findings as I'm poking around in here. Is updating the subgraphs to spit back data uniformly an option for us? Maybe not as part of this PR, but in the near future

Reply:
it turns out that the `useHistoricalRates` query does some parsing under the hood of the rates it fetches. 
- https://github.com/Synthetixio/js-monorepo/blob/%40synthetixio/data%402.46.5-ovm.1/packages/data/src/index.ts#L197
then:
- https://github.com/Synthetixio/js-monorepo/blob/ad1a23c6f03019cfa4e60049eb7a9aff87424dcf/packages/data/queries/rateUpdates/parse.ts#L18

so, for the purpose of this PR, I'm doing the same quick math on the timestamp we get from the `useCandlesticksQuery` as the one that we get from `useHistoricalRates` query, to keep things uniform. 

I'm going to follow up with a PR to have the Kwenta UI use the `useCandlesticksQuery` from the monorepo instead of from `kwenta`, and subsequently delete the query within Kwenta

Reply:
Sounds good. Thanks for digging into this